### DISCUSSION TITLE
Set variable to text file value

### DISCUSSION THREAD

**Discussion by RealMrObvious** [WarmUpTill/SceneSwitcher#970] [Q&A] [answered]
Trying to set a variable to the value of a text file. Was looking at the wiki and have it currently trying to read the value of a text on screen which accesses the file name.txt (equal to `TestName`) but when i try the tutorial in the wiki it ends up giving the variable the value of
`{"read_from_file":true,"file":"pathtofile/name.txt"}`

Looking for any solutions

**Comment by WarmUpTill** [comment, marked as answer]
Something like this should do the tirck:
![image](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/9d1edebb-fd98-45cc-84e1-e0a21dcf21da)

What did you try / which wiki entry are you referring to?

**Reply by RealMrObvious** [reply]
yeah that worked perfectly.
